superchain/implementations/networks/mainnet.yaml

```diff
@@ -4,11 +4,25 @@ l1_erc721_bridge:
   2.1.0: 0xAE2AF01232a6c4a4d3012C5eC5b1b35059caF10d
 l1_standard_bridge:
   2.1.0: 0x64B5a5Ed26DCb17370Ff4d33a8D503f0fbD06CfF
-l2_output_oracle:
-  1.8.0: 0xF243BEd163251380e78068d317ae10f26042B292
 optimism_mintable_erc20_factory:
   1.9.0: 0xE01efbeb1089D1d1dB9c6c8b135C934C0734c846
 optimism_portal:
   2.5.0: 0x2D778797049FE9259d947D1ED8e5442226dFB589
+  3.10.0: 0xe2F826324b2faf99E513D16D266c3F80aE87832B
 system_config:
   1.12.0: 0xba2492e52F45651B60B8B38d4Ea5E2390C64Ffb1
+  2.2.0: 0xF56D96B2535B932656d3c04Ebf51baBff241D886
+dispute_game_factory:
+  1.0.0: 0xc641A33cab81C559F2bd4b21EA34C290E2440C2B
+anchor_state_registry:
+  1.0.0: 0x6B7da1647Aa9684F54B2BEeB699F91F31cd35Fb9
```

Comment:
This looks like the impl address. Shouldn't it be the proxy at `0x18DAc71c228D1C32c99489B7323d441E1175e443`?